### PR TITLE
ADEN-4627 | Adjust Prebid price granularity

### PR DESCRIPTION
**Pull request by mleczey**
## Links
https://wikia-inc.atlassian.net/browse/ADEN-4627
## Description
Adjust Prebid price granularity

### CODE REVIEW

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/prebidSettings.spec.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		{actual: 100.00, expected: '20.00'}
+	];
+
+	cpms.forEach(function (cpm) {
+		it('settings hb_pb function should transform ' + cpm.actual + ' cpm from bidder response to ' + cpm.expected, function () {
+			var settings = prebidSettings.create(),
+				hb_pb = getFunction(settings, 'hb_pb');
+
+			mocks.bidderResponse.cpm = cpm.actual;
+			var actual = hb_pb(mocks.bidderResponse);
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/prebidSettings.spec.js]: let's declare vars at the beginning of scope

**Comment by dianafa** [comment]
Only a nitpick, rest LGTM

File: extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+	/*
+	 * Documentation:
+	 * - http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.setPriceGranularity
+	 * - http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.bidderSettings
+	 * - http://prebid.org/dev-docs/examples/custom-price-bucket.html
+	 */
+	function transformPrice(bidResponse) {
+		var cpm = bidResponse.cpm,
+			result = '20.00';
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js]: why do we want to default to 20.00? maybe null is a better option?

File: extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	'use strict';
+
+	/*
+	 * Documentation:
+	 * - http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.setPriceGranularity
+	 * - http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.bidderSettings
+	 * - http://prebid.org/dev-docs/examples/custom-price-bucket.html
+	 */
+	function transformPrice(bidResponse) {
+		var cpm = bidResponse.cpm,
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js]: is it possible CPM is not set? Maybe we should proof our solution for such case.

File: extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.prebidSettings', [], function () {
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js]: empty array is redundant.

File: extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js (hunk 2 above)
Comment (dianafa) [reply extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js]: then we don't need to make additional if below I guess

**Comment by bognix** [comment]
and you need to fix all the places in code where we read `pbAg` and send it to tracking. Instead we should track what our custom method is returning.

File: extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js (hunk 3 above)
Comment (mleczey) [reply extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js]: discussed offline, documentation and examples say that cpm is always set and number

File: extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js (hunk 2 above)
Comment (mleczey) [reply extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js]: discussed offline, 20.00 is default value for values >= 20

**Comment by bognix** [comment]
LGTM

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**